plaso/cli/status_view.py

```diff
@@ -307,6 +308,25 @@ def _PrintExtractionStatusUpdateWindow(self, processing_status):
       # We need to explicitly flush stdout to prevent partial status updates.
       sys.stdout.flush()
 
+  def _PrintEventsStatus(self, processing_status):
```

Comment:
Just pass the events_status, rather than processing_status as it's not used.

Reply:
ack